### DISCUSSION TITLE
Reward Calculations

### DISCUSSION THREAD

**Discussion by santonode** [FreeLoaderz/rwd-frontend#56] [Ideas]
As an end user, is there a way to see some info on the current reward options available to me? example how many and what are the names of the tokens? What are the qty/epoch options? how long will they be available? how many will each end user receive if they stake to Apex or Panl or Lido? Can I choose from a list of available tokens every epoch? Can we have info on each token? website? policyid? maxsupply? etc?

**Comment by fisherstevenk** [comment]
This is something that @kthedges12 is working on right now. 

He's pulling the list of tokens available on Cardano along with their metadata.

It's on the miro board:  https://miro.com/app/board/uXjVOJ_s9xo=/?moveToWidget=3458764522900164207&cot=14

That doesn't cover your entire list of questions, but it covers a lot of them.

**Comment by kthedges12** [comment]
![metadata](https://user-images.githubusercontent.com/43018628/165156337-b4bcd3d6-b821-4732-853d-5d84577b9a8d.png)
This is what it currently looks like. Thoughts? @santonode . This is the data that is pulled from the Cardano offchain metadata repository. Anymore information would require a different data source.

**Reply by santonode** [reply]
Well yes, but how would I select them to be added to my rewards options? these tokens are available on Testnet? What tokens is SmartClaimz have available today?

**Comment by kthedges12** [comment]
I envisioned it that this “explore” page would have all of the tokens
available on the platform. If you want to see which are available
specifically to you then you can connect your wallet and you will be
presented the available ones.

**Comment by fisherstevenk** [comment]
Keeping with the current theme, please use the blue text for headers and white for the text. 

Hovers will end up being red.

See the historical table for the scheme.

I'd put the policy id and url as viewable text as well. 

Looking good though!

Chris, we can mark tokens as available through smartclaimz and have the as a filter.  That can be the first step to doing what you want to do with this.

**Comment by santonode** [comment]
What is the status on this?

**Comment by Blockjock** [comment]
While searching tokens available, would this allow users to see  other participating “freeloaderz pool” tokens, even though they may not have their wallet staked to that pool?  Also any staking or other incentive based options avail
